_releases/make-release-page.py

```diff
@@ -302,6 +302,16 @@ def generateDownloadInfo(outfile, version, downloadInfo):
     """
     Write out source and binary download links.
     """
+    outfile.write(F"""
+## Binary distributions
+
+| Platform       | Files | Size |
```

Comment:
It would be even better to call out that the package manager might already have it: apt/dnf/brew/conda install root-system or cerm-root might do. (We should lost the actual commands for these four package managers.)

Do you think you could provide this in a follow-up PR? This one is very nice as is!